packages/pigment-css-react/src/utils/generateThemeSource.ts

```diff
@@ -0,0 +1,24 @@
+import type { Theme } from './extendTheme';
+import { generateThemeTokens } from './generateCss';
+
+export function generateThemeSource(theme: Theme | undefined) {
+  if (!theme) {
+    return `export default {}`;
+  }
+  const hasBreakpoints = !!theme.unstable_args?.breakpoints;
+  const hasTransitions = !!theme.unstable_args?.transitions;
+  const breakpointsImport = hasBreakpoints
+    ? "import createBreakpoints from '@mui/system/createTheme/createBreakpoints';\n"
+    : '';
+  const transitionsImport = hasTransitions
+    ? `import createTransitions from '${process.env.PACKAGE_NAME}/createTransitions';\n`
+    : '';
+  const source = `${breakpointsImport}${transitionsImport}
```

Comment:
I think this should come from Material UI. Pigment should just call `theme.toRuntime()` and let Material UI decide what to add to the runtime.

In Material UI, we can export an adapter function that inserts `.toRuntime()` to the theme.

```js
// next.config.js
import { createPigmentConfig, extendTheme } from '@mui/material/styles';

const config = createPigmentConfig(extendTheme());

module.exports = withPigment(config);
```

Reply:
If you agree with my proposal, I can create a PR to add the `createPigmentConfig()`.

Reply:
Agree with having this in Material UI side, and by default Pigment CSS can only add the CSS vars. What do you think?

Reply:
Yes, I think Pigment should call the specific method rather than owning the logic inside.

To let Pigment generate CSS variables from the theme:

> create a method `generateStyleSheets` to `theme`. At build time, Pigment will create CSS file from the result of the method.

To create a runtime theme:

> create a method `toRuntime` to `theme`. At build time, Pigment will use that result and export as JS (the result has to be a string)

To use the runtime theme:

> import theme from '@pigment-css/react/theme';